servers/rendering/renderer_rd/shaders/volumetric_fog.glsl

```diff
@@ -503,7 +503,7 @@ void main() {
 				float shadow_attenuation = 1.0;
 
 				if (d * spot_lights.data[light_index].inv_radius < 1.0) {
-					float attenuation = get_omni_attenuation(d, spot_lights.data[light_index].inv_radius, spot_lights.data[light_index].attenuation);
+					float attenuation = get_omni_attenuation(d, spot_lights.data[light_index].inv_radius, spot_lights.data[light_index].attenuation, omni_lights.data[light_index].max_energy);
```

Comment:
Typo, `omni_lights.data[light_index].max_energy` should likely be `spot_lights.data[light_index].max_energy`.